THANKS.txt

```diff
@@ -130,7 +130,9 @@ Jörg Dietrich for the k-sample Anderson Darling test.
 Blake Griffith for improvements to scipy.sparse.
 Andrew Nelson for scipy.optimize.differential_evolution.
 Brian Newsom for work on ctypes multivariate integration.
-Nathan Woods for advising on multivariate integration and building unit tests.
+Nathan Woods for work on multivariate integration. His work from 31 Aug 2015
+    to the present is subject to the disclaimer at:
+    https://github.com/losalamos/Gazebo/blob/master/LICENSE.
```

Comment:
If the disclaimer is really needed, then I'd rather just include it in this file or in the source code somewhere, rather than use a link that may break.